eng/packaging.targets

```diff
@@ -1,4 +1,16 @@
 <Project InitialTargets="_OverridePackDependsOnForCsProjToPkgProj">
+  
+  <PropertyGroup>
+    <TargetsForTfmSpecificContentInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificContentInPackage);AddRefAssemblies;AddRuntimeSpecificAssemblies;LibIntellisenseDocs</TargetsForTfmSpecificContentInPackage>
+    <TargetsForTfmSpecificDebugSymbolsInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRefDebugSymbolsWithTfm;AddRuntimeDebugSymbolsWithTfm</TargetsForTfmSpecificDebugSymbolsInPackage>
+    <!-- Impossible to satisfy NU5131 for packages containing `ref` - https://github.com/NuGet/Home/issues/8684 -->
+    <NoWarn Condition="'$(IsPackable)' == 'true'">$(NoWarn);NU5131</NoWarn>
+    <IncludeBuildOutput Condition="'$(TargetsAnyOS)' != 'true' or '$(ExcludeFromPackage)' == 'true'">false</IncludeBuildOutput>  
+    <!-- Don't include target platform specific dependencies, since we use the target platform to represent RIDs instead -->
+    <SuppressDependenciesWhenPacking Condition="'$(ExcludeFromPackage)' == 'true' or ('$(TargetsAnyOS)' != 'true' and $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0')))">true</SuppressDependenciesWhenPacking>
+    <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);RemoveRefDuplicates</GenerateNuspecDependsOn>
+  </PropertyGroup>
```

Comment:
I believe all of these properties are specific to if a package is packable. Should the `$(IsPackable)` condition be applied to the entire property group?